azure-pipelines/azure-pipelines.yml

```diff
@@ -536,6 +536,24 @@ stages:
               make buildx-push-image-deviceshifu-http-lwm2m
             displayName: build deviceshifu-http-lwm2m
 
+      - job: docker_push_deviceshifu_gateway_lwm2m
+        steps:
+          - task: GoTool@0
+            inputs:
+              version: $(goVersion)
+          - task: Docker@2
+            displayName: Login to DockerHub
+            inputs:
+              command: login
+              containerRegistry: dockerhub-connection
+          - script: |
+              docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+              docker buildx create --use
+            displayName: configure multi-arch and buildx
+          - script: |
+              make buildx-push-image-deviceshifu-gateway-lwm2m
```

Comment:
what is `buildx-push-image-deviceshifu-gateway-lwm2m`?

Reply:
did you mean this?
https://github.com/Edgenesis/shifu/blob/main/Makefile#L79-L82

Reply:
yes

Reply:
the name is `buildx-push-image-gateway-lwm2m` instead of `buildx-push-image-deviceshifu-gateway-lwm2m`